src/sentry_freight/plugin.py

```diff
@@ -53,6 +53,12 @@ class FreightPlugin(ReleaseTrackingPlugin):
     description = 'Integrate Freight release tracking.'
     version = sentry_freight.VERSION
 
+    def has_project_conf(self):
```

Comment:
Yes, but when I remove it, the Freight plugin doesn't show in the Release Tracking view. Let me see if I can edit the PluginList component to get this to render without that.